bin/runnable-list.js

```diff
@@ -1,40 +1,37 @@
 'use strict'
 
-var program = require('commander')
+const program = require('commander')
 
-var runnable = require('../lib/runnable')
-var Table = require('../lib/table')
-var utils = require('../lib/utils')
+const List = require('../lib/list')
+const Table = require('../lib/table')
+const Utils = require('../lib/utils')
 
 program
   .arguments('[repository]')
   .description('Lists all repositories and services. Specify a repository to list its containers.')
   .parse(process.argv)
 
-var options = {
-  _user: runnable.user,
+const options = {
   repository: program.args.shift()
 }
 
 if (options.repository) {
-  runnable.listContainersForRepository(options)
-    .then(function (results) {
+  List.listContainersForRepository(options)
+    .then((results) => {
       Table.log(results)
     })
-    .catch(utils.handleError)
+    .catch(Utils.handleError)
 } else {
-  runnable.listContainerSummary(options)
-    .then(function (results) {
+  List.listContainerSummary(options)
+    .then((results) => {
       function tableFormatter (obj, cell) {
-        for (var key in obj) {
+        for (const key in obj) {
```

Comment:
why? (this one I can almost see because it changes all the time, but what I'm doing is preventing it from being assigned to in the body of the `for` loop)